cocotb/handle.py

```diff
@@ -782,7 +782,14 @@ def setimmediatevalue(self, value):
         """
```

Comment:
This docstring should get a `.. deprecated:: 1.4.0` entry, and `str` needs to change to `bytes` above